readme.md

```diff
@@ -1,12 +1,12 @@
 [comment]: # "Auto-generated SOAR connector documentation"
 # MaxMind
 
-Publisher: Splunk  
```

Comment:
The change to this `readme.md` is auto generated by `pre-commit run --all-files`.